econml/automated_ml.py

```diff
@@ -285,12 +283,12 @@ def __init__(self, *args, **kwargs):
         # Loop through the kwargs and args if any of them is an AutoMLConfig file, pass them
         # create model and pass model into final.
         new_args = ()
-        for var in args:
+        for idx, arg in enumerate(args):
             # If item is an automl config, get its corresponding
             # AutomatedML Model and add it to new_Args
-            if isinstance(var, EconAutoMLConfig):
-                var = self._get_automated_ml_model(kwarg, key)
-            new_args += (var,)
+            if isinstance(arg, EconAutoMLConfig):
+                arg = self._get_automated_ml_model(arg, f"arg{idx}")
```

Comment:
and what if "arg0" is also a keyword argument?

Reply:
I believe it is just used to make it easier for a person to identify the experiment when looking through experiments in AzureML studio, and that the service itself adds a unique suffix so that the whole thing will be unambiguous in any case.